### PR TITLE
Rebalances some more traitor item costs

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
This lowers some mostly unused, niche or otherwise extremely weak items (Looking at you, feral cats) prices. There are also three cases where the price was increased. Most notable of these are the nuclear medikit and the stealth biochip.

This is a continuation of #21753, but these are all the balance changes of it.

## Why It's Good For The Game
Traitor Items:
Cardboard Cutouts: 1TC `(-80%)`
Cham Voice Mod Mask: 8TC `(+60%)`
Chameleon Flag: 1TC `(-80%)`
Chameleon Stamp: 1TC `(-80%)`
DNA Scrambler: 7TC `(-30%)`
False Bottom Briefcase: 10TC `(-33%)`
Feral Cat Grenade: 10TC `(-33%)`
Singularity Beacon: 10TC `(-67%)`
Stechkin magazine: 3 TC `(-40%)`
Stechkin Special magazine: 6 TC `(-40%)`
Chameleon Thermals: 15 TC `(-50%)`
Stealth bio-chip: 45 TC `(+13%)`
Armored jacket: 3 TC `(-40%)`

Operative Items:
Flamethrower: 3TC `(-40%)`
MedKit: 30TC `(+50%)`
Manhack Delivery: 10TC `(-60%)`
Eshield: 40TC `(-50%)`
Grenade Belt: 120TC `(-20%)`
Assault Borg: 250TC `(-23%)`
Gygax: 400TC `(-11%)`
Mauler: 599TC `(-14%)`

## Testing
Compiled, checked their prices via my uplink.

## Changelog
:cl:
tweak: Some nuclear and traitor items have been reduced in price
/:cl:

### CODE REVIEW

**Comment by Zorazi-Teo-Daedalus** [comment]
You listed "Singularity Beacon" twice.

image

**Comment by DGamerL** [comment]
> You listed "Singularity Beacon" twice.
> 
> image

So bad it had to be reduced twice 😔

**Comment by SpringSkipper** [comment]
for some reason, a mauler being 599 instead of 600 is hilarious.
the only cost change that I question is the reduction to thermals; they seem very strong despite the unfortunate flash weakness.

**Comment by DGamerL** [comment]
> for some reason, a mauler being 599 instead of 600 is hilarious. the only cost change that I question is the reduction to thermals; they seem very strong despite the unfortunate flash weakness.

Do note that the MODsuit thermals area already 15TC!

File: code/datums/uplink_items/uplink_traitor.dm [hunk 1]
```diff
@@ -429,15 +429,15 @@
 	reference = "FRAME"
 	item = /obj/item/cartridge/frame
 	excludefrom = list(UPLINK_TYPE_NUCLEAR, UPLINK_TYPE_SST)
-	cost = 20
+	cost = 15
```
Comment (Coolrune206) [inline code/datums/uplink_items/uplink_traitor.dm:432]:
```suggestion
	cost = 20
```
For reasons discussed in discord, maybe don't reduce the price of this one.

File: code/datums/uplink_items/uplink_nuclear.dm [hunk 2]
```diff
@@ -466,7 +466,7 @@
 	desc = "An incredibly useful personal shield projector, capable of reflecting energy projectiles, but it cannot block other attacks. Pair with an Energy Sword for a killer combination."
 	item = /obj/item/shield/energy
 	reference = "ESD"
-	cost = 80
+	cost = 60
```
[the remark this answers is not in the document]
Comment (DGamerL) [reply code/datums/uplink_items/uplink_nuclear.dm:469]: Description is, will update in a second

**Review by Burzah — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**